server/grpc_bytestream.go

```diff
@@ -326,28 +326,19 @@ func (s *grpcServer) Write(srv bytestream.ByteStream_WriteServer) error {
 	}()
 
 	select {
-	case err, ok := <-recvResult:
-		if !ok {
-			msg := "Receive loop closed unexpectedly."
-			s.accessLogger.Printf("GRPC BYTESTREAM WRITE FAILED: %s", msg)
-			return status.Error(codes.Internal, msg)
-		}
```

Comment:
If I understand correctly, the only change required to this file is an added `pw.CloseWithError(err)` ? If so, please undo the other changes.

Reply:
done